trie/sync_bloom.go

```diff
@@ -110,12 +97,11 @@ func (b *SyncBloom) init(database ethdb.Iteratee) {
 		// If the database entry is a trie node, add it to the bloom
 		key := it.Key()
 		if len(key) == common.HashLength {
-			b.bloom.Add(syncBloomHasher(key))
+			b.bloom.AddHash(binary.BigEndian.Uint64(key))
```

Comment:
Any particular reason for calling binary.Bigendian to manually convert instead of `common.BytesToHash` like other occurances? Seems a bit dangerous as the two methods could go out of sync.

Reply:
The `syncBloomHasher` was replaced by it's `Sum64`, which is `binary.BigEndian.Uint64`

Reply:
My point here was that we have an explicit manual calculation and an implicit built-in calculation now, and it seems dangerous to have two means to arrive to the same number, as eventually one will get modified without the other and things will bork.

Reply:
There might be a misunderstanding here. 

However, we could keep `b.bloom.Add(syncBloomHasher(key))` and redefine `syncBloomHasher` as `binary.BigEndian.Uint64` ? 

But I think you are conflating two different things, the built-in calculation has nothing to do with this